src/main/java/org/spongepowered/api/scoreboard/TeamBuilder.java

```diff
@@ -125,14 +126,14 @@
     TeamBuilder deathTextVisibility(Visibility visibility);
 
     /**
-     * Sets the set of {@link User}s on the {@link Team}.
+     * Sets the set of {@link Teammate}s on the {@link Teammate}.
      *
      * <p>By default, this is the empty set.</p>
      *
-     * @param users The set of {@link User}s on the {@link Team}
+     * @param teammates The set of {@link Teammate}s on the {@link Team}
      * @return This builder
      */
-    TeamBuilder users(Set<User> users);
+    TeamBuilder Teammates(Set<Teammate> teammates);
```

Comment:
the method name should be lowercase `TeamBuilder teammates(Set<Teammate> teammates);`